jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.kt

```diff
@@ -143,6 +143,14 @@ class QueryEditor internal constructor(
         return unit.second
     }
 
+    fun setSelectedTimeUnit(unit: ChronoUnit) {
```

Comment:
Also uh what does this do?

Reply:
the item being set needs to be equal to something else that's currently in the model, which is impossible if the model holds a bunch of pairs and we're trying to select a ChronoUnit